packages/platform/atoms/booker/BookerPlatformWrapper.tsx

```diff
@@ -43,12 +43,18 @@ import { AtomsWrapper } from "../src/components/atoms-wrapper";
 type BookerPlatformWrapperAtomProps = Omit<BookerProps, "username" | "entity"> & {
   rescheduleUid?: string;
   bookingUid?: string;
-  firstName?: string;
-  lastName?: string;
-  guests?: string[];
-  name?: string;
   username: string | string[];
   entity?: BookerProps["entity"];
+  // values for the booking form and booking fields
+  defaultFormValues?: {
```

Comment:
I would call them "bookingFields" to reflect data structure when working with API.